docker/config.ru

```diff
@@ -33,6 +33,8 @@ if ENV.key?('REDIS_SENTINELS')
   end
 end
 
+config[:id] = nil if ENV.fetch('REDIS_NO_CLIENT_NAME', false).to_s.casecomp('true').zero?
```

Comment:
I would recommend changing it to `REDIS_DISABLE_CLIENT_ID` option and also rely only on the presence of the variable (i.e true only if set)

```suggestion
config[:id] = nil if ENV.key?('REDIS_DISABLE_CLIENT_ID')
```

P.S test are broken because of the last commit I made. I'm planning migrate to the GH Actions and will fix specs

Reply:
I updated the PR.